src/mote/templates/mainpage.html

```diff
@@ -72,12 +79,12 @@
             </div>
         </nav>
         <div class="container-lg">
-            <div class="card text-center" style="margin-top: 10%;">
+            <div class="card text-center" style="margin-top: 0%;">
+                <div id="loader"></div>
                 <div class="card-body">
```

Comment:
```suggestion
                <div class="card-body pt-0">
```

It looks like this now.

![image](https://user-images.githubusercontent.com/49605954/147431831-cdd60afc-3854-4310-a582-598ce92760cd.png)

The padding at the top of the calendar header is not needed. We'll remove it and it would look like the following.

![image](https://user-images.githubusercontent.com/49605954/147431871-7e339fc1-6f03-4483-bbac-01971b95e1a3.png)